Anchal_Hora/Calc/calc.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Calculator</title>
+    <script src='https://kit.fontawesome.com/a076d05399.js'></script>
+
+    
+    <link rel="stylesheet" type="text/css" href="./assets/styles/calc.css">
+<!--<script src="./assets/scripts/script.js" ></script> -->
+
+
+</head>
+<body>
+    
+    <div class="container-grid">
+        <div class="output"><form name="calc"><input type="text" name="display" id="scn" size="20" readonly></form></div>
+
+        <div class="buttons">
+            <p>
+                <input type="button" value="x!" class="other_opr" onclick=fact();>
+                <input type="button" value="C" class="other_opr">
+                <input type="button" value="CE" class="other_opr" id="ce_btn" onclick="calc.display.value=''">
+                <button class="major_opr" onclick="calc.display.value+='/'"><i class='fa fa-divide '></i></button>
```

Comment:
Violating Single Responsibility Principle

Reply:
okay!

Reply:
I still see JS in HTML, not resolved